tests/unit/test-cmd/test.build.js

```diff
@@ -86,6 +121,44 @@ describe('build', () => {
     );
   });
 
+  it('throws an error if the filename contains a path', () => {
```

Comment:
Nit, it would probably good to have also one test that verifies that if an interpolated part of the template is a string that looks like a path then we still throw this kind of UsageError (which is actually the case with the version in this patch, it would be mostly an additional test to have more coverage in case of a refactoring), but we can deal with it in a follow up.